.github/workflows/e2e_custom_cl.yml

```diff
@@ -19,13 +19,35 @@ jobs:
         RUSTUP_HOME: "/root/.rustup"
         FORCE_COLOR: 1
     steps:
+      - name: Install latest Git version (>= 2.18.0) for actions/checkout
+        run: |
+          apt-get install software-properties-common -y
+          add-apt-repository ppa:git-core/ppa
+          apt update
+          apt install git -y
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Comment:
needed because the default git version from the container is `2.17` for ubuntu 18.04